### ISSUE TITLE
Confirm revert via the quick Revert button

### ISSUE THREAD

**Issue by Klap-in** [splitbrain/dokuwiki#3202]
I was wondering, the revert button looks like the 'old revisions' button. I already clicked accidentally on a mobile screen the wrong button.

What about adding a confirmation dialog to the revert button? One with focus on the Ok button, such that pressing 'Enter' will confirm it directly.

**Comment by splitbrain** [comment]
Well, I'm happy to improve the used icon. Regarding the confirm dialog... a revert can be reverted so there's no real harm done, is there?

**Comment by Klap-in** [comment]
I'm wondering if everybody actual observes this, if this happens.

**Comment by splitbrain** [comment]
Never happened to me, but I guess I'm the wrong person to ask. Maybe ask in the forum?

**Comment by Klap-in** [comment]
Especially in a tablet view, this can be more an issue. For the desktop view it will show the text labels if you hover, which helps preventing it already. On a tablet you don't have this hover.
It is not a big issue for me, but in this short term of using, I already hit it...

**Comment by phy25** [comment]
I would regard it as a delete action even it's revertable. A quick JavaScript `confirm()` should just work.

**Comment by Michaelsy** [comment]
I totally agree with Klap-in. It has happened to me as well and it took me a really long time to understand what was going on. I was pretty confused. 

I mean, it's not the usual system behavior when you're already making an active change or editing by clicking a button. At least not if it is not obvious from the environment and the label that something is not only called up passively. A symbol alone is usually not sufficient for this. Unless it has a practiced meaning due to its form and position.

In addition, there is the problem that the symbol can easily be confused with the Old Revisions symbol. But a symbol change alone is not enough, IMHO.

\- Michael Sy.

**Comment by splitbrain** [comment]
Fair enough. I guess a confirm dialog won't hurt. PR welcome.

**Comment by Nightfirecat** [comment]
Looks like this should be fixed/closed since #3222 is merged?
